test/src/core/stdlib.js

```diff
@@ -166,10 +166,10 @@ describe('foam.Boolean', function() {
     expect(foam.Boolean.compare(true, true)).toBe(0);
     expect(foam.Boolean.compare(false, false)).toBe(0);
 
-    expect(foam.Boolean.compare(true, 0)).toBe(1);
+    expect(foam.Boolean.compare(true, 0)).toBe(-1);
     expect(foam.Boolean.compare(false, 66)).toBe(-1);
-    expect(foam.Boolean.compare(true, 9)).toBe(0);
-    expect(foam.Boolean.compare(false, 0)).toBe(0);
+    expect(foam.Boolean.compare(true, 9)).toBe(-1);
+    expect(foam.Boolean.compare(false, 0)).toBe(-1);
   });
```

Comment:
Notice that due to the recent changes, `foam.Boolean.compare` is no longer performing a loose comparsion